### PR TITLE
add opensuse, fedora,debian communities repos

### PR DESCRIPTION
**Pull request by jyzhang4834200**
add opensuse, fedora,debian communities repos

### CODE REVIEW

File: communities/Debian.yml [hunk 1]
```diff
@@ -0,0 +1,134 @@
+community_logo_url: "anolisos \u7684\u4ED3\u5E93"
+community_name: anolisos_repos
+community_org_url: ANOLISOS REPOS
```
Comment (eyehwan) [inline communities/Debian.yml:3]: Debian

File: communities/Debian.yml [hunk 2]
```diff
@@ -0,0 +1,134 @@
+community_logo_url: "anolisos \u7684\u4ED3\u5E93"
+community_name: anolisos_repos
```
Comment (eyehwan) [inline communities/Debian.yml:2]: Debian

File: communities/Debian.yml [hunk 3]
```diff
@@ -0,0 +1,134 @@
+community_logo_url: "anolisos \u7684\u4ED3\u5E93"
```
Comment (eyehwan) [inline communities/Debian.yml:1]: Debian

File: communities/fedora-infra.yml [hunk 4]
```diff
@@ -0,0 +1,195 @@
+community_logo_url: "anolisos \u7684\u4ED3\u5E93"
```
Comment (eyehwan) [inline communities/fedora-infra.yml:1]: fedora-infra

File: communities/openSUSE.yml [hunk 5]
```diff
@@ -0,0 +1,644 @@
+community_logo_url: "anolisos \u7684\u4ED3\u5E93"
```
Comment (eyehwan) [inline communities/openSUSE.yml:1]: openSUSE

**Review by eyehwan — CHANGES_REQUESTED** [review]
I think you use the template of anolios without chaning the name.

File: communities/Debian.yml (hunk 2 above)
Comment (jyzhang4834200) [reply communities/Debian.yml]: modified

File: communities/fedora-infra.yml (hunk 4 above)
Comment (jyzhang4834200) [reply communities/fedora-infra.yml]: modified

File: communities/openSUSE.yml (hunk 5 above)
Comment (jyzhang4834200) [reply communities/openSUSE.yml]: modified

**Pull request merged**